packages/x-data-grid-pro/src/tests/columns.DataGridPro.test.tsx

```diff
@@ -55,9 +55,9 @@ describe('<DataGridPro /> - Columns', () => {
       await act(() => apiRef.current?.showColumnMenu('brand'));
       const menu = screen.getByRole('menu');
       await waitFor(() => {
-        expect(menu.id).to.match(/:r[0-9a-z]+:/);
+        expect(menu.id).to.match(/[«|:]r[0-9a-z]+[»|:]/);
       });
-      expect(menu.getAttribute('aria-labelledby')).to.match(/:r[0-9a-z]+:/);
+      expect(menu.getAttribute('aria-labelledby')).to.match(/[«|:]r[0-9a-z]+[»|:]/);
```

Comment:
This test looks very much like we test implementation details 😁

Reply:
Great point! 💯 
@mui/xgrid would it be possible to use the `toHaveAccessibleName` assertion in this case? 🤔